src/components/Location.js

```diff
@@ -0,0 +1,36 @@
+import styled from 'styled-components/macro'
+import PropTypes from 'prop-types'
+
+Location.propTypes = {
+  name: PropTypes.string,
+  image: PropTypes.string,
+  id: PropTypes.number,
```

Comment:
Maybe you dont want to be that precise with the id because you might have id's with characters later on (uuid generate ids with characters for example). `PropTypes.node` would just take anything

Reply:
You're right but for the moment it's fine. Maybe I will use uuid later when I got more content, then I switch to `PropTypes.node`.